doc/source/examples/dem/granular-dam-break/granular-dam-break.rst

```diff
@@ -0,0 +1,218 @@
+
+==================================
+Granular dam-break
+==================================
+
+This example simulates the dam-break flow of a dry granular material using the discrete element method (DEM). More information regarding the DEM parameters are given in the Lethe documentation, i.e. `DEM parameters <../../../parameters/dem/dem.html>`_.
+
+
+Features
+----------------------------------
+- Solvers: ``dem_3d``
+- Three-dimensional problem
+- Moving floating mesh
+- Load-balancing
+
+
+
+Files used in this example
+----------------------------
+
+- Parameters file for the problem (H=20cm): ``examples/dem/3d-dam-break/granular-dam-break.prm``
+- Parameters file for the H=40cm bed: ``examples/dem/3d-dam-break/granular-dam-break.prm``
+- GMSH geo file: ``examples/dem/3d-dam-break/square.geo``
+- GMSH mesh file: ``examples/dem/3d-dam-break/square.msh``
+
+
+
+
+Description of the case
+-----------------------
+
+This example simulates the dam-break flow of a dry powder. First, a packing of particles is generated by inserting particles behind a sluice gate. After 0.5s, the sluice gate rises and the particles are able to flow freely, producing a dam break flow. We compare the results we obtain with those of `Evangelista et al. <https://www.cabdirect.org/cabdirect/abstract/20153319263>`_. To ensure that the simulation remains computationnaly efficient, we will scale-up the sand particles used by Evangelista et al. from a diameter of 1.6mm to 10mm. This reduces their number by a factor of approximately 250x.We first simulate case B1 of Evangelista et al., with a bed length of 60cm and bed height of 20cm. As an extension, we also simulate a bed height of 40cm (case B3).
```

Comment:
```suggestion
This example simulates the dam-break flow of a dry powder. First, a packing of particles is generated by inserting particles behind a sluice gate. After 0.5s, the sluice gate rises and the particles are able to flow freely, producing a dam break flow. We compare the results we obtain with those of `Evangelista et al. <https://www.cabdirect.org/cabdirect/abstract/20153319263>`_. To ensure that the simulation remains computationally efficient, we will scale-up the sand particles used by Evangelista et al. from a diameter of 1.6mm to 10mm. This reduces their number by a factor of approximately 250x. We first simulate case B1 of Evangelista et al., with a bed length of 60cm and bed height of 20cm. As an extension, we also simulate a bed height of 40cm (case B3).
```

Reply:
I'm not sure if the link to the Evangelista et al. reference is working, but I was not able to access it. Maybe you can add the reference in the end as well? As in the previous DEM examples...